src/components/Hero.astro

```diff
@@ -1,4 +1,8 @@
 ---
+interface Props {
+  imageSrc?: string 
+}
+
 const {
   imageSrc = "https://imgs.search.brave.com/ZGKZILBT-Ftxa9snn2h547yBn7VF0sYa8v9mEUDImt0/rs:fit:860:0:0:0/g:ce/aHR0cHM6Ly9pbWcu/ZnJlZXBpay5jb20v/cGhvdG9zLWdyYXR1/aXRlL2hlcm9zLWZv/dS1leHByZXNzaW9u/LWNvbGVyZV8xMTk0/LTQwMjMuanBnP3Nl/bXQ9YWlzX2h5YnJp/ZCZ3PTc0MA"
```

Comment:
@williamgentil this image src should be passed into the called component, not inside the component itself